gpii/configs/all.development.dr.production.txt

```diff
@@ -19,4 +19,5 @@ This mean that:
 
 Notes on testing:
 This setup is used as a basis for the platform specific acceptance tests that make use of a dynamic device reporter.
-Since the acceptance tests might use slightly different paths for the tests (ie. a special folder for reading prefs and solutionsRegistry), what is actually being used for testing is a slight variation of this config. For more details, see the txt files describing the configs in the acceptance tests folder of both universal and the platform specific repositories.
+Since the acceptance tests might use slightly different paths for the tests (ie. a special folder for reading prefs and solutionsRegistry), 
+what is actually being used for testing is a slight variation of this config. For more details, see the txt files describing the configs in the acceptance tests folder of both universal and the platform specific repositories.
```

Comment:
All our acceptance tests are in the universal repo, there's nothing to check in platform-specific repos